src/pages/instances/actions/DeleteInstanceBtn.tsx

```diff
@@ -34,24 +33,20 @@ const DeleteInstanceBtn: FC<Props> = ({ instance }) => {
   const isDisabled = isLoading || instance.status !== "Stopped";
 
   return (
-    <Tooltip
-      message={isDisabled ? "Instance must be stopped" : undefined}
-      position="btm-center"
-    >
+    <div className="delete-instance-btn-wrapper">
       <ConfirmationButton
-        className="u-no-margin--bottom"
+        toggleAppearance="base"
+        className="delete-instance-btn u-no-margin--bottom"
```

Comment:
We don't need the margin-bottom class here, instead I'd move the margins from the `delete-instance-btn-wrapper` to the `delete-instance-btn`. It should simplify the whole bit in here.